### DISCUSSION TITLE
Follow Up Bound Interstitial Concentration of Compound

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#1194] [Questions & Problems]
Hello, I'm following up on a previous question about what the albumin bound concentration of a compound would be in the brain interstitial space, knowing what the unbound fraction is. Is the concentration of albumin in the brain interstitial space available in PK-Sim or Mobi, and would this help in determining the bound fraction?

**Comment by StephanSchaller** [comment]
You can best look into this when looking at the model in MoBi. The different PC methods use different values for the tissue/interstitial albumin fractions/amounts.
The container/water partitioning value for interstitial brain will give you the answer on the fraction unbound.

Best,
Stephan
